testnet/src/node.rs

```diff
@@ -50,6 +59,12 @@ struct RegisteredKey {
     vrf_public_key: VRFPublicKey,
 }
 
+enum RelayerDirective {
+    HandleNetResult(NetworkResult),
+    ProcessTenure(BurnchainHeaderHash, BurnchainHeaderHash, BlockHeaderHash),
+    RunTenure(RegisteredKey, BurnchainTip),
```

Comment:
Cool! I think this is also paving support for microblocks (we would add an entry `ProcessMicroblock` or so)